website/src/pages/index.module.css

```diff
@@ -10,6 +10,11 @@
   overflow: hidden;
 }
 
+.heroBanner a {
+  color: var(--ifm-hero-text-color);
```

Comment:
As I don't know out of my head what `--ifm-hero-text-color` resolves to, it would be nice to mention the new color in the commit message.